docs/en/core-services/Catalog.md

```diff
@@ -102,43 +102,219 @@ $ ${ALLUXIO_HOME}/bin/alluxio table attachdb --db alluxio_db hive \
-        "tables": [
-          "table1",
-          {"table": "table2", "partitions": ["table2_part1", "table2_part2"]}
-        ]
-      }
-    }
-    ```
-
-    You can specify which tables and partitions within these tables should be bypassed from Alluxio.
-    By specifying only the table name, all partitions of that table, if any, will be bypassed. 
-    Otherwise, you can specify specific partitions to bypass.
+1. Bypassing tables and partitions:
+
+   You can specify some tables and partitions to be bypassed from Alluxio, so that data will not be
+   cached in the Alluxio file system, instead computing frameworks will be directed to access data directly from the UFS.
+   This can be helpful when some tables and partitions are large, and accommodating them in the cache
+   is undesirable. An example configuration is like the following:
+
+   ```json
+   {
+     "bypass": {
+       "tables": {
+         "include": [
+           {"type": "name", "name": "table1"},
+           {"type": "regex", "regex": "table[23]"},
+           {
+             "type": "partition_spec",
+             "table": "table4", 
+             "partitions": {
+               "include": [
+                 {"type": "name", "name": "table4_part1"}, 
+                 {"type": "regex", "regex": "table4_part[23]"}
+               ]
+             }
+           }
+         ]
+       }
+     }
+   }
+   ```
+
+   There are 2 ways to specify which tables and partitions within these tables should be 
+   bypassed from Alluxio.
     
-    In the example above, table 1 is fully bypassed. Partition 1 and 2 of table 2 are bypassed, 
-    and any other partitions, if any, are not.
-
-> **Note:** When databases are attached, all tables are synced from the configured UDB.
-If out-of-band updates occur to the database or table and the user wants query results to reflect
-the updates, the database must be synced. See [Syncing Databases](#syncing-databases) for more
-information. 
+   1. Plain names: exact names of tables or partitions;
+   2. Regular expressions: patterns to capture a set of table or partition names. The patterns
+      are matched as a whole on the names, i.e., it is as if the patterns were enclosed in a pair 
+      of `^` and `$`.
+
+   Additionally, for tables, you can specify only some partitions of a table to be bypassed, 
+   and the rest to be mounted normally. 
+   
+   If a table is specified by its exact name or captured by a pattern, all partitions of the table, 
+   if any, are bypassed. In other words, the table is "fully" bypassed.
+    
+   In the above example, table 1, 2, and 3 are fully bypassed.
+   Partition 1, 2, and 3 of table 4 are bypassed, and any other partitions, if any, are not.
+   
+2. Ignoring tables:
+
+   You can specify some tables to be ignored when attaching the database to Alluxio. Compared to 
+   bypassed tables, Alluxio will not try to mount the ignored tables, therefore they are 
+   invisible to the Alluxio client. 
+   
```

Comment:
Provide use cases for ignored tables, i.e. when this feature is useful, similar to the documentation about bypassed tables above.

Reply:
@HelloHorizon @apc999 opinions?

Reply:
LGTM